docs/src/index.md

```diff
@@ -525,11 +525,165 @@ To reference columns with more complicated expressions, you must wrap column ref
 @transform df :a + $(get_column_name(x))
 ```
 
-## Constructing multi-column arguments and `src => fun => dest` calls using `$`
+## Operations with multiple columns at once using `AsTable` inside operations
 
-If an argument is entirely wrapped in `$()`, the result bypasses the anonymous function creation of DataFramesMeta.jl and is passed to the underling DataFrames.jl function directly. Importantly, this allows for multi-column selection and passing `src => fun => dest` calls from the DataFrames.jl "mini-language" directly.
+In operations, it is also allowed to use `AsTable(cols)` to work with
+multiple columns at once, where the columns are grouped together in a
+`NamedTuple`. When `AsTable(cols)` appears in a operation, no
+other columns may be referenced in the block.
```

Comment:
and it can appear only once - right?

Reply:
No. it can appear multiple times. But it has to look the exact same, you can't have, for example

```
@transform df :y = sum(AsTable([:a, :b])) + sum(AsTable(["a", "b"]))
```

There are similar restrictions elsewhere, for example with `@astable`.